CHANGELOG.rst

```diff
@@ -41,6 +41,33 @@ Fixed
 -----
 - ``rasa test nlu`` with a folder of configuration files
 
+Fixed
+-----
+
+
+[1.2.4] - 2019-08-23
+^^^^^^^^^^^^^^^^^^^^
+
+Added
+-----
```

Comment:
let's remove all these empty ones and just keep the `Fixed`

Reply:
oooops

Reply:
should've done that in the released version 😬